feature/detail/src/main/java/team/ppac/detail/DetailScreen.kt

```diff
@@ -0,0 +1,189 @@
+                shape = FarmemeRadius.Radius20.shape,
+            ),
+    ) {
+        Column(
+            modifier = Modifier.padding(10.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            AsyncImage(
+                model = ImageRequest
+                    .Builder(LocalContext.current)
+                    .data(TEST_IMAGE_PREVIEW)
+                    .crossfade(true)
+                    .build(),
+                placeholder = painterResource(R.drawable.detail_sample),
+                contentDescription = null,
+                contentScale = ContentScale.Crop,
+                modifier = Modifier.clip(FarmemeRadius.Radius10.shape),
+            )
+            DetailTexts()
+            DetailFunnyButton()
+        }
+    }
+}
+
+@Composable
+fun DetailTexts() {
+    Spacer(modifier = Modifier.height(25.dp))
+    Text(
+        text = "나는 공부를 찢어",
+        color = FarmemeTheme.textColor.primary,
+        style = FarmemeTheme.typography.heading.large.semibold,
+    )
+    Spacer(modifier = Modifier.height(5.dp))
+    Text(
+        text = "#공부 #학생 #시험기간 #힘듦 #피곤",
+        color = FarmemeTheme.textColor.tertiary,
+        style = FarmemeTheme.typography.body.large.medium,
+    )
+    Spacer(modifier = Modifier.height(11.dp))
+    Text(
+        text = "출처: 출처에 대한 내용이 들어갑니다.",
+        color = FarmemeTheme.textColor.assistive,
+        style = FarmemeTheme.typography.body.xSmall.medium,
+    )
+    Spacer(modifier = Modifier.height(20.dp))
+}
+
+@Composable
+fun DetailFunnyButton() {
+    Button(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 10.dp)
+            .padding(bottom = 10.dp)
+            .height(46.dp),
+        colors =
+            ButtonDefaults.buttonColors(
+                backgroundColor = FarmemeTheme.skeletonColor.primary,
+            ),
+        elevation = null,
+        onClick = {},
+    ) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center,
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_kk),
+                contentDescription = null,
+            )
+            Spacer(modifier = Modifier.width(6.dp))
+            Icon(
+                painter = painterResource(id = R.drawable.ic_funny),
+                contentDescription = null,
+            )
+        }
+    }
+}
+
+@Composable
+fun DetailBottomBar() {
+    TabBar(modifier = Modifier.fillMaxWidth()) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .align(Alignment.CenterVertically)
+                .padding(horizontal = 20.dp),
+        ) {
+            DetailBottomButton(
+                icon = { FarmemeIcon.Copy(modifier = Modifier.size(20.dp)) },
+                title = "복사",
+            ) {
+            }
```

Comment:
onClickButton을 람다로 이렇게 여는거 보단 위처럼 하는게 더 좋을 것 같아용
```suggestion
            onClickButton = ~
```

Reply:
@JaesungLeee  혹시 이유가있다면 뭘까용? 후행람다 적극 사용하는게 좋은거 아닐지!
```
 .clickable(onClick = onClickButton),
```
```
.clickable{ onClickButton() }
```
```
composeBox(
){
onClickButton()
}
```
```
fun Modifier.clickable(
    enabled: Boolean = true,
    onClickLabel: String? = null,
    role: Role? = null,
    onClick: () -> Unit
) = composed(
```

우리가 저번에 람다생성 비용때문에 이렇게 하기로했던거 때문인건가?
저것도 후행람다에서 {} 바로 본문쓸수 있는 걸 전달해서 주긴하넹! 

랍다생성 비용이 왜 발생하는지 이해를 잘못했응..{ } 후행람다가 좋다고했는데.. 좀더 찾아봐야겠다!

Reply:
일단 수정완룡 ~ 리뷰감사해용!